README.md

```diff
@@ -81,14 +81,41 @@ constructs the case class specified in its type constraint. To do this,
 building inner validated case class instances or from using the
 `validate` or `unchecked` methods to validate values at a given path.
 
-### Copy-free Validated `Config` Instances
+### Secure Validated `Config` Instances
 
 Sometimes, after validating and building a `Config` instance, we do not
 wish the constructed case class instance to be modified (e.g. by calling
 a `copy` constructor) - allowing this could allow validation guarantees
 to be broken! To support such scenarios, one simply extends the trait
 `CopyFreeConfig` when defining the case class.
 
+Secure configuration objects may then be constructed as follows:
+```scala
+object SecureConfigBuilder {
+  abstract case class SecureHttpConfig private[SecureConfigBuilder] (host: String, port: Int) extends CopyFreeConfig[SecureSettings]
+  abstract case class SecureSettings private[SecureConfigBuilder] (name: String, timeout: FiniteDuration, http: HttpConfig) extends CopyFreeConfig[SecureSettings]
+
+  def apply(filename: String): SecureSettings = {
+    validateConfig(filename) { implicit config =>
+      build[SecureSettings](
```

Comment:
Builders here need to call the class constructor and not (as they currently do) the companion object's apply.